lib/rules/no-object-create.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @author Yosuke Ota
+ * See LICENSE file in root directory for full license.
+ */
+"use strict"
+
+const { ReferenceTracker, READ } = require("eslint-utils")
+
+module.exports = {
+    meta: {
+        docs: {
+            description: "disallow the `Object.create` method.",
+            category: "ES5",
```

Comment:
If you release it in a minor version, we need to change it to `undefined` so that it doesn't affect the ruleset.